packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -296,11 +333,20 @@ class Shortcuts extends StatefulWidget {
   @override
   _ShortcutsState createState() => _ShortcutsState();
 
+  // Returns a human-readable description of the shortcut key map.
+  String _debugShortcutMapDescription() {
```

Comment:
This looks like a nice improvement. I would suggest implementing a
ShurtcutProperty or LogicalKeySetMapProperty
You can see an example of a custom property here:
https://github.com/flutter/flutter/blob/5005824763e1e581f3afede54bb9c85a9c42f8c5/packages/flutter/lib/src/painting/matrix_utils.dart#L547

Reply:
Ooh, excellent idea.  I added `ShortcutMapProperty`.